src/app/client/src/app/modules/content-search/components/view-all/view-all.component.ts

```diff
@@ -432,7 +432,12 @@ export class ViewAllComponent implements OnInit, OnDestroy, AfterViewInit {
       const enrolledCourseCount = _.get(courseData, 'enrolledCourses').length;
       this.noResult = false;
       this.totalCount = enrolledCourseCount;
-      this.searchList = this.formatSearchresults(_.orderBy(_.get(courseData, 'enrolledCourses'), ['enrolledDate'], ['desc']));
+      const sortedData = _.map(_.orderBy(_.get(courseData, 'enrolledCourses'), ['enrolledDate'], ['desc']), (val) => {
+        const value = _.get(val, 'content');
+        value['mimeType'] = 'application/vnd.ekstep.content-collection';
```

Comment:
This should not be hardcoded. We have to take it mimetype from val.mimetype if it available.

Reply:
Done.